containers/autoReply/AutoReplyModal.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+    Modal,
+    ContentModal,
+    FooterModal,
+    Button,
+    PrimaryButton,
+    useMailSettings,
+    useApiWithoutResult,
+    useEventManager
+} from 'react-components';
+import { updateAutoresponder } from 'proton-shared/lib/api/mailSettings';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import AutoReplyForm from './AutoReplyForm/AutoReplyForm';
+import useAutoReplyForm from './AutoReplyForm/useAutoReplyForm';
+
+const AutoReplyModal = ({ onClose }) => {
+    const [{ AutoResponder }] = useMailSettings();
+    const { model, updateModel, toAutoResponder } = useAutoReplyForm(AutoResponder);
+    const { request } = useApiWithoutResult(updateAutoresponder);
+    const { call } = useEventManager();
+
+    const handleSubmit = async () => {
+        await request(toAutoResponder(model));
+        call();
```

Comment:
We should display a notification to say "Auto-reply updated". For notification you can use `useNotifications`.

``` js
const { createNotification } = useNotifications();
createNotification({ text: c('Success').t`Auto-reply updated` });
```